thunder/clang/__init__.py

```diff
@@ -1799,6 +1799,13 @@ def logical_and(a, b):
     return a & b
 
 
+@clangop()
```

Comment:
I know this is a bit of a pain, but would you separate `logical_not` into it's own PR, @k223kim? More focused PRs are easier to review and merge

Reply:
That makes sense! Will do that.